components/collective-page/index.js

```diff
@@ -166,8 +164,6 @@ class CollectivePage extends Component {
   renderSection(section) {
     switch (section) {
       // v2 transition - these standalone sections will be removed from render if feature flag is active
-      case Sections.GOALS:
-        return <SectionGoals collective={this.props.collective} />;
```

Comment:
Do we want to remove that now? It will be an inconvenient change for some users and my understanding of the latest conversations in https://github.com/opencollective/opencollective/issues/3698 was that there was non-breaking way of implementing this:

```es6
settings: {
  collectivePage: {
    sections: [
      { type: 'SECTION', section: 'GOALS' },
      { 
        type: 'SECTION_CATEGORY', 
        category: 'CONNECTED',
        sections: [
          { section: CONVERSATIONS, isEnabledSettingKey: 'features.conversations.enabled' },
        ]
      }
  ]
}
```

Reply:
Yeah, there's lots of conversations going on in a few issues, a bit hard to keep up. You're right, I think I'll remove changes to GOALS from this PR and deal with that separately.